### PR TITLE
Refactor color schema

### PR DESCRIPTION
**Pull request by ivan-bonk**
add services for Sizes
add services for Color

### CODE REVIEW

File: modules/color/color.service.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    const color = await Color.findByIdAndDelete(id);
+    if (!color) {
+      throw new Error(COLOR_NOT_FOUND);
+    }
+    return color;
+  }
+
+  async updateColor(id, input) {
+    const color = await Color.findById(id);
+    if (!color) {
```
Comment (moran711) [inline modules/color/color.service.js:38]: You can check if color with `colorHex `already exist in update method.

File: modules/product/product.model.js [hunk 2]
```diff
@@ -17,7 +16,12 @@ const productSchema = new mongoose.Schema({
   innerMaterial: [Language],
   strapLengthInCm: Number,
   images: PrimaryImage,
-  colors: [Color],
+  colors: [
```
Comment (moran711) [inline modules/product/product.model.js:19]: We have only one color in product, so don't use array.

**Review by moran711 — COMMENTED** [review]
Please add permission file to colors and sizes.

**Review by Iryna-Bzdyr — APPROVED** [review]

**Review by moran711 — APPROVED** [review]

**Review by moran711 — APPROVED** [review]

**Review by ahd3r — APPROVED** [review]

File: error-messages/sizes.messages.js [hunk 3]
```diff
@@ -1,5 +0,0 @@
-const SIZES_NOT_FOUND = 'SIZES_NOT_FOUND';
```
Comment (dimaPalii) [inline error-messages/sizes.messages.js:1]: It's better to rename file name instead of creating new one

File: modules/color/color.resolver.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+const colorService = require('./color.service');
+
+const colorQuery = {
+  getAllColors: async () => colorService.getAllColors(),
+
+  getColorById: async (_, { id }) => {
```
Comment (dimaPalii) [inline modules/color/color.resolver.js:6]: Do we really need '_' here?

File: modules/size/size.resolver.js [hunk 5]
```diff
@@ -15,4 +15,39 @@ const sizeQuery = {
   },
 };
 
-module.exports = { sizeQuery };
+const sizeMutation = {
+  addSize: async (_, { data }) => {
```
Comment (dimaPalii) [inline modules/size/size.resolver.js:19]: Same here with `_`

**Review by dimaPalii — COMMENTED** [review]
Approve after fix comments

**Pull request merged**